worker/worker.go

```diff
@@ -258,16 +258,20 @@ func (w *Worker) EnsureRunningWatchers(schedulerNames []string) {
 		} else {
 			var occupiedTimeout int64
 			var configYaml *models.ConfigYAML
+			var gameName string
+
 			configYamlStr, err := models.LoadConfig(w.DB, schedulerName)
 			if err == nil {
 				configYaml, err = models.NewConfigYAML(configYamlStr)
 				if err == nil {
 					occupiedTimeout = configYaml.OccupiedTimeout
+					gameName = configYaml.Game
 				}
 			}
 			if err != nil {
 				l.Warnf("error loading scheduler %s: %s", schedulerName, err.Error())
 				occupiedTimeout = w.Config.GetInt64("occupiedTimeout")
+				gameName = w.Config.GetString("game")
```

Comment:
configYaml has an attribute Game

Reply:
yes, I'm using it in the upper block (err == nil). I thought that this block was a fallback, so I repeated the existing behavior from the line above it, reading from w.Config. Is it wrong?